tests/CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ if(GEN_FILES)
     add_custom_target(tls13-compat.sh
         DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/opt-testcases/tls13-compat.sh)
     set_target_properties(tls13-compat.sh PROPERTIES EXCLUDE_FROM_ALL NO)
+    add_dependencies(${ssl_opt_target} tls13-compat.sh)
```

Comment:
Actually wait, shouldn't we do something similar for make as well?

Reply:
Good point! I'd only thought about it from the point of view of having already run `make generated_files`, which doesn't exist with CMake. But I do expect all make targets to work from a fresh checkout. Fixed.